doc/nrf/known_issues.rst

```diff
@@ -1320,6 +1351,38 @@ nrfxlib
 Crypto
 ======
 
+.. rst-class:: v1-9-0
+
+NCSDK-13825: Mbed TLS legacy APIs from Oberon has limited TLS/DTLS support
+  The legacy Mbed TLS APIs in Oberon for TLS/DTLS do not support the RSA and DHE algorithms.
+
+.. rst-class:: v1-9-0
+
+NCSDK-13841: Limited support for RSA in PSA APIs
+  The provided RSA implementation in the PSA APIs has limited support in accelerators.
+  Only the CryptoCell accelerator supports RSA in PSA APIs. Currently, the only supported mode is PKCS1-v1.5. The key size needs to be smaller than 2048 bits and the supported hash functions are SHA-1/SHA-224/SHA-256.
+
+.. rst-class:: v1-9-0
+
+NCSDK-13843: Limited support for MAC in PSA APIs
+  The provided message authentication codes (MAC) implementation in the PSA APIs has limited support in accelerators. Only the CryptoCell accelerator supports MAC operations in PSA APIs and the supported hash algorithms are SHA-1/SHA-224/SHA-256.
+
+.. rst-class:: v1-9-0
+
+NCSDK-13843: Limited support for key derivation in PSA APIs
+  The provided key derivation implementation in the PSA APIs has limited support in accelerators. Only the CryptoCell accelerator supports key derivation in PSA APIs and the supported hash algorithms are the SHA-1/SHA-224/SHA-256.
+
+.. rst-class:: v1-9-0
+
+NCSDK-13844: Limited support for GCM in PSA APIs in Oberon
+  The provided GCM implementation of the PSA APIs in the Oberon accelerator only supports 12 bytes IV.
+
+.. rst-class:: v1-9-0
+
+NCSDK-13842: Limited ECC support in PSA APIs
+  The provided ECDSA implementation in the CryptoCell accelerator does not support 521 bit curves.
+  The provided ECDSA implementation in Oberon only supports the curves secp224r1 and secp256r1.
```

Comment:
Remove this line (on oberon only supporting curve secp224r1 and secp256r1), this is not a "known issue". This is a limitation of feature which is described in details in configuration...

Reply:
Removed this.